src/2048-expert/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="description" content="A challenging version of 2048. Start your advanture!">
+    <meta name="author" content="Zihan Zhong">
```

Comment:
I think you can remove this. Alternatively, all the HTML files can have `<meta name="author" content="KKDZ">` but we will need to discuss this within the team.

Reply:
Ok, I will remove it for now